.travis.yml

```diff
@@ -50,9 +50,8 @@ install:
   - sudo make install
 script:
   - echo "build successful"
-  - echo "cd ~/ngxpagespeed"
-  - echo "sudo ./test/run_tests.sh 8050 8051 $HOME/mod_pagespeed/src $HOME/nginx-branches-default/objs selfsigned.modpagespeed.com"
-  - echo "sudo ./test/run_tests.sh 8050 8051 $HOME/mod_pagespeed $HOME/nginx-branches-default/objs/nginx selfsigned.modpagespeed.com"
+  - cd ~/ngxpagespeed
+  - sudo ./test/run_tests.sh $HOME/mod_pagespeed $HOME/nginx-branches-default/objs/nginx
```

Comment:
> I don't follow the update.py comment. If that's breaking all our Travis builds, we should fix it.

I'm not actually sure what broke Travis, but it looks like it was about two days ago.  I'll look at that.

> I think we should (1) fix that and (2) do the mps submodule thing before "uncorking" travis.

The main reason Travis fails is because of build errors, where an ngx_pagespeed change depends on a mod_pagespeed change that's under review.  This is separate from the problem where we've never actually been running ngx_pagespeed's tests on Travis.

Regardless, this isn't critical to this CL, just something I wanted to clean up while I was in there, so I've changed it back to an echo.